
# [CVE-2020-13596](https://docs.djangoproject.com/en/3.0/releases/security/)

## Description

- `An issue was discovered in Django 2.2 before 2.2.13 and 3.0 before 3.0.7. Query parameters generated by the Django admin ForeignKeyRawIdWidget were not properly URL encoded, leading to a possibility of an XSS attack.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 4.3

## Scores

- **Exploitability Score**:
  - 8.6
- **Impact Score**:
  - 2.9
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2020-06-03 14:15:12
- **Vulnerability Status**:
  - Modified

## References

- **cve@mitre.org**: https://docs.djangoproject.com/en/3.0/releases/security/
- **cve@mitre.org**: https://groups.google.com/forum/#%21msg/django-announce/pPEmb2ot4Fo/X-SMalYSBAAJ
- **cve@mitre.org**: https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/4A2AP4T7RKPBCLTI2NNQG3T6MINDUUMZ/
- **cve@mitre.org**: https://security.netapp.com/advisory/ntap-20200611-0002/
- **cve@mitre.org**: https://usn.ubuntu.com/4381-1/
- **cve@mitre.org**: https://usn.ubuntu.com/4381-2/
- **cve@mitre.org**: https://www.debian.org/security/2020/dsa-4705
- **cve@mitre.org**: https://www.djangoproject.com/weblog/2020/jun/03/security-releases/
- **cve@mitre.org**: https://www.oracle.com/security-alerts/cpujan2021.html
